examples/ui_action_examples/public/components/README.md

```diff
@@ -0,0 +1,5 @@
+# Presentation React components
```

Comment:
Do you think we need this `layer` of components? Are we plan to reuse them outside of the drilldown? 
Otherwise I'd just keep it: 
`public/dashboard_to_discover_drilldown/components/**` and not expose it

Reply:
Done.